hexrd/ui/utils.py

```diff
@@ -255,3 +255,8 @@ def exclusions_off(plane_data):
         yield
     finally:
         plane_data.exclusions = prev
+
+
+def has_nan(x):
+    # Utility function to check if there are any NaNs in x
+    return np.isnan(np.min(x))
```

Comment:
@joelvbernier Yeah, I would have used a numpy function such as `np.any_nan()` if there was one. But there's not, so I had to make a utility function.

Using `np.isnan(x)` can be a problem if `x` is a very big array, because it creates another array of booleans of the same size. For instance, a  100k x 100k array would create and return a 100k x 100k array of booleans.

And unfortunately, we can't do `np.nan in x`, since `np.nan != np.nan`.

`np.min(x)` avoids creating another array of the same size, so it has good performance. See some discussion [here](https://stackoverflow.com/questions/6736590/fast-check-for-nan-in-numpy). There are other possible solutions there such as `np.isnan(np.sum(x))`.

Putting it into a utility function helps when the implementation is not pretty.